homeassistant/components/demo/__init__.py

```diff
@@ -279,26 +281,94 @@ async def _insert_statistics(hass):
     )
     async_add_external_statistics(hass, metadata, statistics)
 
-    # Fake yesterday's energy consumption
+    # Add external energy consumption in kWh, ~ 12 kWh / day
+    # This should be possible to pick for the energy dashboard
+    statistic_id = f"{DOMAIN}:energy_consumption"
     metadata = {
         "source": DOMAIN,
-        "statistic_id": f"{DOMAIN}:energy_consumption",
+        "name": "Energy consumption (kWh)",
```

Comment:
Why add unit in name?

Reply:
It's there to aid frontend debugging & development, see https://github.com/home-assistant/frontend/pull/13454